### PR TITLE
feat(chatgpt): add theme

### PR DESCRIPTION
**Pull request by adriankarlen**
I couldn't get highlight.js to work using imports so I ended up copying the hljs code to the css, unsure if this has something to do with how the hljs port of rose pine is setup?

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -10,14 +10,18 @@
...
 
 # Userstyles
+
 **Productivity**
-- [GitHub](https://github.com/rose-pine/userstyles/tree/main/github)
-- [Proton](https://github.com/rose-pine/userstyles/tree/main/proton)
+
+-   [ChatGPT](https://github.com/rose-pine/userstyles/tree/main/chatgpt)
+-   [GitHub](https://github.com/rose-pine/userstyles/tree/main/github)
+-   [Proton](https://github.com/rose-pine/userstyles/tree/main/proton)
```
Comment (mvllow) [inline readme.md:27]: Please keep the same list formatting

File: readme.md [hunk 2]
```diff
@@ -10,14 +10,18 @@
 </p>
 
 # Getting Started
+
 1. Install the [Stylus](https://github.com/openstyles/stylus) extension for [Chrome](https://chrome.google.com/webstore/detail/stylus/clngdbkpkpeebahjckkjfobafhncgmne) or [Firefox](https://addons.mozilla.org/en-GB/firefox/addon/styl-us/).
 2. Downloading Userstyles:
-   - Open the folder of the userstyle you want to install.
-   - Then, click on the "Stylus Install" button or open `rose-pine.user.css` and click on "Raw."
+    - Open the folder of the userstyle you want to install.
+    - Then, click on the "Stylus Install" button or open `rose-pine.user.css` and click on "Raw."
```
Comment (mvllow) [inline readme.md:17]: Please keep the same list formatting

File: chatgpt/readme.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+1. Make sure you have the [Stylus](https://github.com/openstyles/stylus) browser extension installed, then visit [this link](https://github.com/rose-pine/userstyles/raw/main/github/rose-pine.user.css)
+2. Configure your preferred accent color and theme variant
+3. Click install on the top left
+
+## Gallery
+
+![Rosé Pine](assets/deafult.webp)
+![Rosé Pine Moon](assets/moon.webp)
+![Rosé Pine Dawn](assets/dawn.webp)
```
Comment (mvllow) [inline chatgpt/readme.md:25]: Please use the GitHub CDN or another image host for these so they don't need to be part of the repo (:

**Review by mvllow — CHANGES_REQUESTED** [review]
Sorry this took me a while to get to. Thank you for the new theme!

**Comment by adriankarlen** [comment]
Resolved the comments now.

**Review by mvllow — APPROVED** [review]

**Pull request merged**